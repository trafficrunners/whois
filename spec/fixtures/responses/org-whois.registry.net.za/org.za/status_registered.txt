Domain Name: joburg.org.za
Domain ID: dom_8VP-9999
WHOIS Server: org-whois2.registry.net.za

Updated Date: 2015-02-05T08:45:51Z
Creation Date: 1997-10-03T09:46:34Z
Registry Expiry Date: 2999-12-31T21:59:59Z
Sponsoring Registrar: ZA Central Registry
Sponsoring Registrar IANA ID: 9999 
Domain Status: ok

Registrant ID: jobuRant
Registrant Name: City of Johannesburg Metropolitan Municipality
Registrant Organization: 
Registrant Street: P.O. Box 30757  
Registrant City: Braamfontein
Registrant State/Province: Gauteng
Registrant Postal Code: 2017
Registrant Country: ZA
Registrant Phone: +27.110186314
Registrant Phone Ext: 
Registrant Fax: +27.113819583
Registrant Fax Ext: 
Registrant Email: joelsonp@joburg.org.za

Admin ID: zacr-a0c0379446
Admin Name: Joelson Pholoha
Admin Organization:  
Admin Street: Private Bag X10013, Sandton, 2146  
Admin City: -
Admin State/Province: 
Admin Postal Code: 
Admin Country: --
Admin Phone: +27.110186314
Admin Phone Ext: 
Admin Fax: +27.113819583
Admin Fax Ext: 
Admin Email: Joelsonp@Joburg.org.za

Billing ID: zacr-07de5cca59
Billing Name: 
Billing Organization:  
Billing Street:   
Billing City: -
Billing State/Province: 
Billing Postal Code: 
Billing Country: --
Billing Phone: 
Billing Phone Ext: 
Billing Fax: 
Billing Fax Ext: 
Billing Email: 

Tech ID: zacr-71fff5bce2
Tech Name: Eben Jacobs
Tech Organization:  
Tech Street: Accounts Payable, Vida Building, Kabelweg 57, 1014 BA Amsterdam  
Tech City: -
Tech State/Province: 
Tech Postal Code: 
Tech Country: --
Tech Phone: +27.110186314
Tech Phone Ext: 
Tech Fax: +27.113819583
Tech Fax Ext: 
Tech Email: ebenj@joburg.org.za


Name Server: demeter.is.co.za
Name Server: jupiter.is.co.za
Name Server: titan.is.co.za
DNSSEC: unsigned
>>> Last update of WHOIS database: 2015-06-03T12:14:13Z <<<

#    WHOIS lookup made at 2015-06-03T12:14:13Z
# --
# The use of this Whois facility is subject to the following terms and
# conditions. https://registry.net.za/whois_terms
# Copyright (c) ZACR 1995-2015
