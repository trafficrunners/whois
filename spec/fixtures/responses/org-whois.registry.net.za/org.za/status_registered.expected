#domain
  %s == "joburg.org.za"

#domain_id
  %s == "dom_8VP-9999"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{1997-10-03 09:46:34 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2015-02-05 08:45:51 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2999-12-31 21:59:59 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "9999"
  %s.name         == "ZA Central Registry"
  %s.organization == "ZA Central Registry"
  %s.url          == nil

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].id            == "jobuRant"
  %s[0].name          == "City of Johannesburg Metropolitan Municipality"
  %s[0].organization  == ""
  %s[0].address       == "P.O. Box 30757"
  %s[0].city          == "Braamfontein"
  %s[0].state         == "Gauteng"
  %s[0].zip           == "2017"
  %s[0].country       == nil
  %s[0].country_code  == "ZA"
  %s[0].phone         == "+27.110186314"
  %s[0].fax           == "+27.113819583"
  %s[0].email         == "joelsonp@joburg.org.za"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "zacr-a0c0379446"
  %s[0].name          == "Joelson Pholoha"
  %s[0].organization  == ""
  %s[0].address       == "Private Bag X10013, Sandton, 2146"
  %s[0].city          == "-"
  %s[0].zip           == ""
  %s[0].country       == nil
  %s[0].country_code  == "--"
  %s[0].phone         == "+27.110186314"
  %s[0].fax           == "+27.113819583"
  %s[0].email         == "Joelsonp@Joburg.org.za"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  %s[0].id            == "zacr-71fff5bce2"
  %s[0].name          == "Eben Jacobs"
  %s[0].organization  == ""
  %s[0].address       == "Accounts Payable, Vida Building, Kabelweg 57, 1014 BA Amsterdam"
  %s[0].city          == "-"
  %s[0].zip           == ""
  %s[0].country       == nil
  %s[0].country_code  == "--"
  %s[0].phone         == "+27.110186314"
  %s[0].fax           == "+27.113819583"
  %s[0].email         == "ebenj@joburg.org.za"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil


#nameservers
  %s %CLASS{array}
  %s %SIZE{3}
  %s[0] %CLASS{nameserver}
  %s[0].name == "demeter.is.co.za"
  %s[0].ipv4 == nil
  %s[0].ipv6 == nil
  %s[1] %CLASS{nameserver}
  %s[1].name == "jupiter.is.co.za"
  %s[1].ipv4 == nil
  %s[1].ipv6 == nil
  %s[2] %CLASS{nameserver}
  %s[2].name == "titan.is.co.za"
  %s[2].ipv4 == nil
  %s[2].ipv6 == nil
